docroot/themes/humsci/humsci_basic/src/scss/ckeditor/imports.scss

```diff
@@ -264,15 +264,24 @@ $px-only: true;
   blockquote:nth-child(n) {
     @include hb-blockquote;
     overflow: visible;
-
-    font-size: 24px;
+    font-size: 24px !important;
```

Comment:
I think some of these `!important` can be removed. For example, the only reason the the `!important` for setting the font size to `28px` in larger screens is because this line also has `!important`. We should try to avoid using `!important` as much as possible.